aikau/src/main/resources/alfresco/renderers/AvatarThumbnail.js

```diff
@@ -112,7 +162,20 @@ define(["dojo/_base/declare",
             this.currentItem[this.userNameProperty] = encodeURIComponent(this.currentItem[this.userNameProperty]);
             this.thumbnailUrlTemplate = "slingshot/profile/avatar/{" + this.userNameProperty + "}/thumbnail/avatar";
          }
+         
          this.inherited(arguments);
+
+         if (this.groupProperty && this.groupValues)
+         {
+            var groupPropertyValue = lang.getObject(this.groupProperty, false, this.currentItem);
+            this.isGroup = array.some(this.groupValues, function(value) {
+               return value === groupPropertyValue;
+            });
+         }
+         if (this.isGroup)
+         {
+            this.thumbnailUrl = require.toUrl("alfresco/renderers/css/images/" + this.groupImage);
```

Comment:
The current code is consistent with the how the folder images are handled in Thumbnail so I'm happy to leave this as is.